modules/account-lib/src/coin/baseCoin/blsKeyPair.ts

```diff
@@ -0,0 +1,75 @@
+import * as BLS from '@chainsafe/bls';
+import { KeyPairOptions, isPrivateKey } from './iface';
+import { BaseKeyPair } from './baseKeyPair';
+import { AddressFormat } from './enum';
+import { NotImplementedError } from './errors';
+
+/**
+ * Base class for BLS keypairs.
+ */
+export abstract class BlsKeyPair implements BaseKeyPair {
+  protected keyPair: BLS.Keypair;
+  protected source?: KeyPairOptions;
```

Comment:
Do you need this field? I don't see you ever set or read from it

Reply:
I was assuming some upstream tools may use it (thought it was a standard object for doing things like getting a keypair from privkey, etc.). It's included in the unit tests as well - can remove though if desired?

Reply:
It's not even set to a value anywhere currently, though

Reply:
I don't see it in the unit tests either. Fields shouldn't be part of the interface - callers can access everything they need via the interface functions. I think you can just remove it no problem